### PR TITLE
metrics: fix 2 incorrect grafana expression

### LINKED ISSUE #15859: tikv's worker pending tasks under task result is not correct
### What version of TiKV are you using?

nightly

### Steps to reproduce

The current expression is "sum(rate(tikv_futurepool_pending_task_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", name=~\"sched-worker-.*\"}[1m]))" but `tikv_futurepool_pending_task_total` is gauge, do rate on gauge is not correct.

### PR DESCRIPTION
**Pull request by glorv**
### What is changed and how it works?

Issue Number: Close #15859

What's Changed:

```
- Fix the incorrect expression on `tikv_futurepool_pending_task_total ` by changing the aggregation function from `rate` to `avg_over_time`
- Fix grafana relate to `tikv_coprocessor_request_wait_seconds_bucket` by add "type=all"
```

Result:
before this change, when running oltp_write_only with 50 threads on a 8c tikv:
image
after this change:
image

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by bufferflies — APPROVED** [review]

File: metrics/alertmanager/tikv.accelerate.rules.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ groups:
     - record: tikv_coprocessor_request_duration_seconds:1m
       expr: sum(rate(tikv_coprocessor_request_duration_seconds_bucket{instance=~".*"}[1m])) by (le)
     - record: tikv_futurepool_pending_task:1m
-      expr: sum(rate(tikv_futurepool_pending_task_total{instance=~".*"}[1m])) by (name)
+      expr: sum(avg_over_time(tikv_futurepool_pending_task_total{instance=~".*"}[1m])) by (name)
```
Comment (Connor1996) [inline metrics/alertmanager/tikv.accelerate.rules.yml:51]: what's the difference

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by Connor1996 — APPROVED** [review]

**Pull request merged**